### ISSUE TITLE
Report feedback

### ISSUE THREAD

**Issue by ksedivyhaley** [UBC-MDS/DSCI_522_Group-308_Used-Cars#38]
- Description of analysis needs clarification. For example:

> "For each model we performed 5-fold-cross-validated grid search involving a range of most important model-specific hyper-parameters."

Brief explanation of why this method is appropriate.

> "Since training and validating took a lot of resources, we performed it on a gradually increasing subsets of training data."

You increased the subset for each model, or you tested a few times for each model with the same, increasing subsets?

> "Since SVM shown the best results from the very beginning, we performed a thorough adaptive grid search on a bigger subset of 200,000 observations (running for 4 hours) resulting in 81.3% accuracy on validation data."

You just showed a table with an 0.813 validation score. It looks like this should be the "very beginning" result, but if this is the case then you should have a different number for the bigger subset.

> "Eg., having an RMSE almost twice higher than MAE suggests that there is a good number of observations where the error is big (the more RMSE differs from MAE, the higher is the variance)"

What are the values for RMSE and MAE?

- Missing some elements from writing
  - INPUT TRAIN SCORE and INPUT TEST SCORE (in report and README)
  - Report has a space where it looks like you intended to link to the scripts (this would be very useful if inserted)! 
- "Given that the dataset was imbalanced, this led to poor prediction of the classes that were quite sparse." - an additional sentence explaining or giving an example would be useful.
- Good description of future work
- Try connecting your work back to the research question (how to predict car prices) more frequently. You do this well in some places ("This is something we may want to improve by finding features and clusters in data space that introduce more variance in the predictions. Eg. the model predicting clean car price may greatly differ from the model predicting salvage (damage / total loss) car price.") and the report would benefit from more consistent application of this.
- Frequent grammar/spelling errors, eg "users that creates vast markets" and "difficult to guage"
